name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/SantanderConsumerBankPDFExtractor.java

```diff
@@ -256,6 +256,31 @@ private void addAccountStatementTransaction()
                             t.setNote(TextUtil.replaceMultipleBlanks(v.get("note")).trim());
                         })
 
+                        .wrap(TransactionItem::new));
+
+        // 20.05.2023 20.05.2023 -1,00 33,01 Auszahlung auf ...
+        Block removalBlock = new Block(
+                        "^(?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) ([\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<amount>[\\.,\\d-]+) ([\\.,\\d-]+) (Auszahlung) (?<note>.*)");
+        type.addBlock(removalBlock);
+        removalBlock.set(new Transaction<AccountTransaction>()
+
+                        .subject(() -> {
+                            AccountTransaction entry = new AccountTransaction();
+                            entry.setType(AccountTransaction.Type.REMOVAL);
+                            return entry;
+                        })
+
+                        .section("date", "amount", "note")
+                        .match("^(?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) ([\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) (?<amount>[-\\.,\\d-]+) ([\\.,\\d-]+) (Auszahlung) (?<note>.*)")
```

Comment:
- Improve regulare expressions
```suggestion
                        .match("^(?<date>[\\d]{2}\\.[\\d]{2}\\.[\\d]{4}) [\\d]{2}\\.[\\d]{2}\\.[\\d]{4} \\-(?<amount>[\\.,\\d]+) [\\.,\\d]+ Auszahlung (?<note>.*)$")
```